File: src/main/java/com/qa/cinema/service/VenueService.java
package com.qa.cinema.service;

public interface VenueService{
	
	String getAllVenues();
	String addNewVenue(String venueJson);
	String updateVenue(Long venue_ID, String updatedVenue);
	String deleteVenue(Long venue_ID);
	String findVenueByID(Long venue_ID);
}

File: src/main/webapp/app/service/venueservice.js
(function(){

    var venueService = function(venueDal){
        this.getAllVenues = function() {
            return venueDal.getAllVenues();
        };

        this.getVenueByID = function(venue_ID) {
            return venueDal.getVenueByID(venue_ID);
        }
    };

    angular.module('qaCinemas2').service('venueService', ['venueDal', venueService]);

}());